README.md

```diff
@@ -49,7 +49,7 @@ app.post('/graphql', graphqlHTTP({
 
 app.get('/graphql', graphqlHTTP({
   schema: MyGraphQLSchema,
-  graphiql: true // set it to false to disable the GraphiQL interface
```

Comment:
I would expect people to understand what `graphiql` false does, but I also believe people will just copy/paste the code example provided without necessarily realising that option can be turned to false if they don't need GraphiQL.